OpenTabletDriver.Plugin/HPETDeltaStopwatch.cs

```diff
@@ -11,7 +11,7 @@ public HPETDeltaStopwatch(bool startRunning = true)
             start = isRunning ? internalWatch.Elapsed : default;
         }
 
-        public static TimeSpan RunitmeElapsed => internalWatch.Elapsed;
+        public TimeSpan RunitmeElapsed => internalWatch.Elapsed;
```

Comment:
`internalWatch` is static, there is no need for `HPETDeltaStopwatch` instance to query app uptime
```suggestion
        public static TimeSpan RunitmeElapsed => internalWatch.Elapsed;
```

Reply:
Ah, this change was not intended. It was when I'm debugging a weird relative mode bug where it jumps even after skipping one, or two reports.